<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DMWidget</class>
 <widget class="QWidget" name="DMWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1536</width>
    <height>864</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_add">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>添加</string>
       </property>
       <property name="iconSize">
        <size>
         <width>14</width>
         <height>14</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_modify">
       <property name="text">
        <string>编辑</string>
       </property>
       <property name="iconSize">
        <size>
         <width>14</width>
         <height>14</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_delete">
       <property name="text">
        <string>删除</string>
       </property>
       <property name="iconSize">
        <size>
         <width>14</width>
         <height>14</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_refresh">
       <property name="text">
        <string>刷新</string>
       </property>
       <property name="iconSize">
        <size>
         <width>14</width>
         <height>14</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
